web/src/components/config_render/ConfigInput.jsx

```diff
@@ -101,7 +100,6 @@ export default class ConfigInput extends React.Component {
             ) : null}
             <div className="field-input-wrapper flex alignItems--center u-marginTop--15">
               <InputField
-                ref={this.inputRef}
```

Comment:
`<InputField>` is now a functional component and doesn't use or provide this ref